app/src/main/res/values-fa/arrays.xml

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- Arrays used by the UI -->
+    <string-array name="aspect_ratio_names">
+        <item>اصلی</item>
+        <item>Panscan</item>
+        <item>۱۶:۹</item>
+        <item>۱۶:۱۰</item>
+        <item>۴:۳</item>
+        <item>۲.۳۵:۱</item>
+    </string-array>
+    <string-array name="aspect_ratios" translatable="false">
```

Comment:
please remove the entries with `translatable="false"`, no need to duplicate them.